source/includes/administration/_email_settings.md

```diff
@@ -0,0 +1,171 @@
+`smtpPassword`<br/>*string* | If your SMTP host requires authentication, specify the password associated with the username you specified.
+`smtpPort`<br/>*integer* | SMTP port number, usually 25 for SMTP or 465 for SMTPS.
+`smtpUseSsl`<br/>*boolean* | Set to true if your SMTP host uses the SSL protocol.
+`mailgunDomain`<br/>sString* | Domain name for the MailGun service.
+`mailgunPrivateKey`<br/>*string* | Private key for the MailGun service.
+
+
+<!-------------------- LIST EMAIL SETTINGS -------------------->
+
+#### List email settings
+
+`GET /email_settings`
+
+```shell
+# Retrieve email settings
+curl "https://cloudmc_endpoint/rest/email_settings" \
+   -H "MC-Api-Key: your_api_key"
+```
+> The above command returns JSON structured like this:
+
+```json
+{
+    "data": [
+        {
+            "mailgunPrivateKey": "VALIDPRIVATEKEY",
+            "smtpHost": "valid.host",
+            "smtpFromAddress": "valid.address@domain.com",
+            "smtpUsername": "valid.username@gmail.com",
+            "organization": {
+                "id": "fb1b2caf-4ca4-4c45-b094-faa84bdabc47"
+            },
+            "smtpPort": 2,
+            "mailgunDomain": "valid.mailgun.domain",
+            "smtpPassword": "coolPassword",
+            "smtpUseSsl": false,
+            "id": "dc30f9a6-1edc-11eb-92b6-0242ac120003"
+    }
+  ]
+}
+```
+List the email settings configured on the system.
+
+Query Params | &nbsp;
+---- | -----------
+`organization_id`<br/>*UUID* | Return only settings for the provided organization id. If not provided, will default to the user organization.
+
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The id of email settings.
+`organization.id`<br/>*UUID* | The organization id that the email settings is linked to.
+`smtpHost`<br/>*string* | Hostname or IP address of your SMTP mail server (e.g. smtp.yourcompany.com).
+`smtpFromAddress`<br/>*string* | Email address used in the 'sender address' (or 'from') field.
+`smtpUsername`<br/>*string* | If your SMTP host requires authentication, specify the username of these authentication credentials here.
+`smtpPassword`<br/>*string* | If your SMTP host requires authentication, specify the password associated with the username you specified.
+`smtpPort`<br/>*integer* | SMTP port number, usually 25 for SMTP or 465 for SMTPS.
+`smtpUseSsl`<br/>*boolean* | Set to true if your SMTP host uses the SSL protocol.
+`mailgunDomain`<br/>sString* | Domain name for the MailGun service.
+`mailgunPrivateKey`<br/>*string* | Private key for the MailGun service.
+
+
+<!-------------------- UPDATE EMAIL SETTINGS -------------------->
+
+#### Update email settings
+
+`PUT /email_settings/:id`
+
+```shell
+# Updates an existing email settings for an organization
+curl -X PUT "https://cloudmc_endpoint/rest/email_settings/:id" \
+   -H "MC-Api-Key: your_api_key"
+```
+
+> Request body example:
+
+```json
+{
+    "organization": {
+		"id": "ce77d759-c015-4a92-b594-3f7119867e1f"
```

Comment:
too many tabs

Reply:
to correct

Reply:
I don't know why it looks like that here. In my editor it looks normally indented 
![image](https://user-images.githubusercontent.com/25533000/98556161-beefcc00-2270-11eb-9478-5e6346c93996.png)

Reply:
space versus tabs....

Reply:
a4adb593d61c7c3fa65fde6a0f98bc502082e75e

Reply:
the id was not fixed

Reply:
```suggestion
    "id": "ce77d759-c015-4a92-b594-3f7119867e1f"
```